src/Pangraph/Containers.hs

```diff
@@ -0,0 +1,36 @@
+module Pangraph.Containers
+    (  toContainerGraph
```

Comment:
I think you should call this function simply `convert`. 

Then you'll have `GraphML.parse`, `VHDL.write`, `Containters.convert`, which is a more consistent pattern.